src/components/Table/TableBody/TableBody.tsx

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
 
+import { CellContext, cellContext } from '../TableCell/TableCell';
+
 type Props = {
   className?: string;
   children?: React.ReactNode;
 };
 
 const TableBody: React.FC<Props> = ({ className, children }: Props) => {
-  return <tbody className={className}>{children}</tbody>;
+  const cellContextValues: CellContext = {
+    location: 'body',
+  };
+
+  return (
+    <tbody className={className}>
+      <cellContext.Provider value={cellContextValues}>{children}</cellContext.Provider>
```

Comment:
A react component is pascal cased

Reply:
It's not a react component, it's a context. But is just checked and the React documentation also pascal cases context, so will change.